ci/ci/constants.py

```diff
@@ -23,23 +22,4 @@
     'GreatBrando',
     'johnc1231',
     'iitalics'
-}   
-
-profiles = {
```

Comment:
er this change wasn't intentional I think? You also got lost the authorized users addition

Reply:
Yeah, I figured I would just issue a separate PR for that.  Most recent commit to this was 30 minutes ago and removed this section?

Reply:
oh, yeah, weird...

Reply:
maybe let's try squashing locally and force-pushing? will come show you how